packer/ansible/rackhd_source.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Development Server
+  hosts: local
+  sudo: no
+  roles:
+    - apt
+    - build
+    - git
+    - rabbitmq
+    - mongodb
+    - node
+    - isc-dhcp-server
+    - monorail
```

Comment:
Based on packer/ansible/roles/monorail/tasks/main.yml , it appears that the monorail role sets up the monorail.json config file.